setup.py

```diff
@@ -1,46 +1,11 @@
-from pathlib import Path
-
-from setuptools import find_packages, setup
-
-import versioneer
```

Comment:
This line should go but the rest must be re-add or, we need a pyproject.toml with these info. I would not do that in the same PR just to keep it a bit simpler for now.